src/helpers/subgraph/__tests__/ProtocolMetricsHelper.unit.test.ts

```diff
@@ -1,5 +1,6 @@
 import { ChartType } from "src/components/Chart/Constants";
 import { getDataKeysFromTokens, getMaximumValue } from "src/helpers/subgraph/ProtocolMetricsHelper";
+import { describe, expect, test } from "vitest";
```

Comment:
haha this is shocking to me every time I see a test that doesn't import these but still works. there's a lot like this.